roles/ispmail-dovecot/tasks/main.yml

```
@@ -63,41 +63,45 @@
     backrefs: true
   notify: restart dovecot
 
-- name: Copy SQL authentication configuration (auth-sql.conf.ext)
+- name: Copy dynamic configuration files
   template:
-    src: auth-sql.conf.ext.j2
-    dest: /etc/dovecot/conf.d/auth-sql.conf.ext
-  notify: restart dovecot
-
-- name: Copy mailbox and namespace configuration (10-mail.conf)
-  copy:
-    src: 10-mail.conf
-    dest: /etc/dovecot/conf.d/10-mail.conf
+    src: "{{ item }}.j2"
+    dest: "/etc/dovecot/conf.d/{{ item }}"
   notify: restart dovecot
+  loop:
+    - 10-mail.conf
+    - 10-ssl.conf
+    - auth-sql.conf.ext
 
-- name: Copy service configuration (10-master.conf)
+- name: Copy static configuration files
   copy:
-    src: 10-master.conf
-    dest: /etc/dovecot/conf.d/10-master.conf
+    src: "{{ item }}"
+    dest: "/etc/dovecot/conf.d/{{ item }}"
```

The quotes are not neccesary there.

---

I don't really get it, what would you want to do ?

---

You could just write
```suggestion
    dest: /etc/dovecot/conf.d/{{ item }}
```
But this is purely aesthetic

---

Ugh, what the hell ? Do you have documentation about that ?

---

If the value in a key/value mapping starts with a `{`, YAML assumes it to be the begining of an  inline array, so we must quote values that begin with a variable enclosed in `{{` for it to be interpreted correctly.
But if there is any other caracter prior to the variable, the quotes are no longer necessary, as YAML won't try to interpret the `{` as the begining of an array.
So this is ok:
```yaml
src: files/{{ inventory_hostname }}
```
But this is not (and should be quoted):
```yaml
dest: {{ install_path }}/Build
```

---

The only related documentation I could find is [this](https://yaml.org/spec/1.2/spec.html#}%20end%20flow%20mapping//) but may not be very helpful..

---

Wow, thanks. I thought substitution was only done in double quoted strings. All of this makes more sense now !